### PR TITLE
refacto trash can

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-model-interface/pull/109
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/413
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1033

### CODE REVIEW

**Comment by itkg-ngilain** [comment]
:+1:

File: BackofficeBundle/Manager/NodeManager.php [hunk 1]
```diff
@@ -165,20 +165,20 @@ protected function getEditableStatus(NodeInterface $node = null)
     public function deleteTree($nodes)
```
Comment (nicolasThal) [inline BackofficeBundle/Manager/NodeManager.php]: did you update the tests for this method?

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**